examples/inference.ipynb

```diff
@@ -1879,6 +1901,262 @@
         "    print_prompt_and_generated(p, g)\n",
         "    print()"
       ]
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "### Perceiver AR symbolic audio modeling\n",
+        "\n",
+        "The following section demonstrates how to use a Perceiver AR audio model to generate symbolic ([MIDI](https://en.wikipedia.org/wiki/MIDI)) audio data. The model has `134M` parameters and was trained with an input context of `4096` prefix tokens and `2048` latent tokens for `27` epochs (`157M` tokens per epoch) on the [GiantMIDI-Piano](https://github.com/bytedance/GiantMIDI-Piano) dataset. \n",
```

Comment:
- ~~The following~~ This section
- a~~n input~~ context

For all further occurrences of "input context" or "input prompt" I'd remove "input"